_posts/intro/2014-02-01-raspberry-pi-quick-start.md

```diff
@@ -39,7 +39,7 @@ in order to enable it follow the instructions available at the following URL:
 
 ## Direct Connection of the Raspberry Pi to the Computer
 
-If you don't use Raspberry Pi with a monitor and keyvbord, you can direct connection of the Raspberry Pi to the Computer.
+If you don't use Raspberry Pi with a monitor and keyvboard, you can direct connection of the Raspberry Pi to the Computer.
```

Comment:
hi @pasqualenaclerio , there is still typo in word "keyvboard", instead you can type "keyboard"